### DISCUSSION TITLE
unprocessable entity

### DISCUSSION THREAD

**Discussion by ghost** [postalserver/postal#1809] [Help with using Postal] [answered]
hi trying to setup postal but i created an org, that worked but seting up email server just shows![Build new mail server - martin - Postal(15:59_22-01-16)](https://user-images.githubusercontent.com/86638691/149649119-0d4693f9-240c-4998-ae2b-efde5639fc19.jpg) i checked with developer tools and just got {
  "status": 422,
  "error": "Unprocessable Entity"
} not sure which logs to check

**Comment by ghost** [comment, marked as answer]
nevermind turned out to be a mysql permission error

**Reply by willpower232** [reply]
glad you figured it out, did you have to go into the container and look at the logs?

**Comment by ghost** [comment]
Actually no I haven't yet as it's a test server and I ended up using root and were trying to get it working with nethserver as a proof of concept (there's nothing else on the server and would obviously be fixing the issue before putting it in production) everything works with root but not with postal@% with all permissions on postal database

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by willpower232** [reply]
Ah you can see in the docs that it also needs permissions for `postal-*` (databases starting with `postal-`). Creating a new mailserver will create a new database called `postal-1` or something like that.

https://docs.postalserver.io/install/prerequisites#mariadb

**Comment by ghost** [comment]
Thanks that makes more sense

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
